### DISCUSSION TITLE
Results Interpretation xPore 2.0

### DISCUSSION THREAD

**Discussion by Wardale24** [GoekeLab/xpore#100] [Q&A] [answered]
Hi @ploy-np,

Could you give us some more insight on how to interpret the results from `xPore 2.0`? The info in #30 is helpful, but now you have given us the possibility of not including an unmodified or knockout sample. 

Without providing a KO or unmodified sample: for one NNANN, if there is a methylation on A in all treatments, is it correct that we would not be able to identify the modified site using `xPore`?

From our understanding, we can see that in certain DRACH motifs we have differential methylation across treatments, but with the lack of a demethylated sample we were wondering if we can identify whether treatment X has a modified site and treatments Y and Z do not have a modified site, or if it were the other way around. 

When it comes to `mod_assignment`, could you give us more information on how to interpret `lower`or `higher`? 

Thank you in advance!

**Comment by yuukiiwa** [comment, marked as answer]
Hi Alex,

Sorry for the late reply!

Yes, `xpore` will not be able to identify the modification site if the same site is methylated across all treatments. To identify the non-differentially expressed m6A sites, we suggest running `m6anet` (https://github.com/GoekeLab/m6anet), which can detect m6A directly from a single sample, on all your samples and compare the m6anet results from there.

Still, it is possible to identify that a site is differentially modified in treatment X but not treatments Y and Z, for xpore does every pairwise comparison between the treatments (X vs Y, X vs. Z, and Y vs. Z). You will have to see whether the differential methylation magnitude `diff_mod_rate_<condition1>_vs_<condition2>` is large enough and whether it is significant based on the `pval_<condition1>_vs_<condition2>`.

A “lower” `mod_assignment` means the mean of the modified distribution is lower than the mean of the unmodified distribution while a “higher” `mod_assignment` means that the mean of the modified distribution is higher than the mean of the unmodified distribution. We can consider only one modification type per k-mer by finding the majority mod_assignment of each k-mer. For example, the majority of the modification means of GGACT (`mu_mod`) is lower than the non-modification counterpart (`mu_unmod`). We can filter out those positions whose `mod_assigment` values are not in line with those of the majority in order to restrict ourselves with one modification type per kmer in the analysis. This can be done by running` xpore postprocessing`.

Best wishes,
Yuk Kei

**Comment by Wardale24** [comment]
Hello @yuukiiwa! 

We are familiar with `m6anet` and will follow your instructions to run it on individual samples! Thank you for this and for clearing up the meaning behind `mod_assignment`!

When it comes to differentially modified between treatments, yes, that is what we initially understood. However, when you refer to whether the differential methylation magnitude between conditions being large enough, is this in reference to what you state in your publication "we first selected those differentially modified sites with effect size >0.5 and P value <0.001"? When you refer to effect size, is it `diff_mod_rate_<condition1>_vs_<condition2>`?

Kind regards,
Alex
